test/unit_tests/graphql/test_processes.py

```diff
@@ -254,10 +318,11 @@ def test_processes_filtering_with_invalid_filter(
                     "status",
                     "workflow",
                     "creator",
-                    "organisation",
+                    "customerId",
```

Comment:
Another comment about customer ;)

Feel free to ignore; there might be other tickets that scope the removal of customer from core.

Reply:
yea, removed it 👍